### PR TITLE
10 feb

### PR DESCRIPTION
**Pull request by MarianBastiurea**
new classes with setter and getter

### CODE REVIEW

File: src/main/java/org/example/c01/gettingStarted/Notes.txt [hunk 1]
```diff
@@ -35,7 +35,18 @@ Operand1 ? Operand2 : Operand3, evaluate Operand1 if it's true will execute Oper
 if it's false will execute Operand3
 a class could be described as :
     a special block code that contains methods
-An object is called an instace of particular clas
+An object is called an instance of particular clas
```
Comment (andreiursudev) [inline src/main/java/org/example/c01/gettingStarted/Notes.txt:38]: typo clas

**Review by andreiursudev — APPROVED** [review]

File: src/main/java/org/example/c01/gettingStarted/Notes.txt [hunk 2]
```diff
@@ -35,7 +35,21 @@ Operand1 ? Operand2 : Operand3, evaluate Operand1 if it's true will execute Oper
...
+Class can be organized into logical groups, which are called packages
```
Comment (andreiursudev) [inline src/main/java/org/example/c01/gettingStarted/Notes.txt:39]: Separate notes per chapter.

File: src/main/java/org/example/c01/gettingStarted/Notes.txt (hunk 1 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c01/gettingStarted/Notes.txt:38]: done

File: src/main/java/org/example/c01/gettingStarted/Notes.txt (hunk 2 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c01/gettingStarted/Notes.txt:39]: done

**Pull request merged**